package.json

```diff
@@ -102,18 +102,18 @@
     "@guardian/src-link": "^3.5.0",
     "@guardian/src-radio": "^3.5.0",
     "@guardian/src-text-input": "^3.5.0",
-    "aws-sdk": "2.893.0",
+    "aws-sdk": "2.900.0",
```

Comment:
I presume this won't trigger the TC issue we had the other day because now we're specifying it as a dep?

Reply:
Yeah, still not entirely sure what the issue was, but now that we specify `@guardian/node-riffraff-artifact` as a dependency the build passes in TeamCity.